src/App/Action/Plugin/Authentication.php

```diff
@@ -86,19 +86,19 @@ class Authentication
      * @param \Magento\Framework\Data\Collection\ModelFactory $modelFactory
      * @param \Magento\Framework\Controller\Result\RedirectFactory $resultRedirectFactory
      * @param \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
-     * @param \Vaimo\AdminAutoLogin\Model\Config\Source\AdminUser $adminUserSource
+     * @param \Vaimo\AdminAutoLogin\Model\Config\Source\AdminUsernameList $adminUsernameList
      * @param AdminSessionsManager $adminSessionsManager
      */
     public function __construct(
-        \Magento\Backend\Model\Auth $auth,
-        \Magento\Backend\Model\UrlInterface $backendUrl,
-        \Magento\Framework\Event\ManagerInterface $eventManager,
-        \Magento\Framework\Message\ManagerInterface $messageManager,
-        \Magento\Framework\Data\Collection\ModelFactory $modelFactory,
-        \Magento\Framework\Controller\Result\RedirectFactory $resultRedirectFactory,
-        \Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig,
-        \Vaimo\AdminAutoLogin\Model\Config\Source\AdminUser $adminUserSource,
-        \Magento\Security\Model\AdminSessionsManager $adminSessionsManager
+        \Magento\Backend\Model\Auth                                 $auth,
+        \Magento\Backend\Model\UrlInterface                         $backendUrl,
+        \Magento\Framework\Event\ManagerInterface                   $eventManager,
+        \Magento\Framework\Message\ManagerInterface                 $messageManager,
+        \Magento\Framework\Data\Collection\ModelFactory             $modelFactory,
+        \Magento\Framework\Controller\Result\RedirectFactory        $resultRedirectFactory,
+        \Magento\Framework\App\Config\ScopeConfigInterface          $scopeConfig,
+        \Vaimo\AdminAutoLogin\Model\Config\Source\AdminUsernameList $adminUsernameList,
+        \Magento\Security\Model\AdminSessionsManager                $adminSessionsManager
```

Comment:
Not a fan of the fancy indentation, it's a pain to upkeep

Reply:
Hmmm.. maybe we should change the whole styling of the code anyway.